ci-operator/step-registry/fips-check/optional-operator-images-scan/fips-check-optional-operator-images-scan-commands.sh

```diff
@@ -0,0 +1,194 @@
+            image_list=$(echo $image_list | sed -r 's/registry.stage.redhat.io/brew.registry.stage.redhat.io/g')
+        fi
+        for image_url in ${image_list}; do
+            imges_test_result=$(ssh -o UserKnownHostsFile=/dev/null -o IdentityFile="${SSH_PRIV_KEY_PATH}" -o StrictHostKeyChecking=no ${BASTION_SSH_USER}@${BASTION_IP} \
+                "sudo podman run --privileged -it -v /tmp/tmp/:/root/:ro registry.ci.openshift.org/ci/check-payload scan operator --spec $image_url" || true)
+            echo "###test result for $image_url is: $imges_test_result" >> $images_test_result
+        done
+        echo "cat ${imges_test_result}" >> ${scan_result_file}
+        resFail=$(grep -E "Failure Report" ${images_test_result} || true)
+        resSuessWithWarnings=$(grep -E "Successful run with warnings" ${images_test_result} || true)
+        if [[ -n $resFail ]];then
+            echo "Fips check  for csv $csv in package $package failed!"
+            echo "$csv" >> ${scan_result_failure_csvs}
+            pass=false
+        elif [[ -n $resSuessWithWarnings ]];then
+            echo "Fips check  for csv $csv in package $package succeed with warnings!"
+            echo "$csv" >> ${scan_result_succeed_with_warnings_csvs}
+            pass=false
+        else
+            echo "Fips check for csv $csv in package $package succeed!"
+        fi
+        #delete containers and images to save disk space
+        echo "delete containers"
+        ssh -o UserKnownHostsFile=/dev/null -o IdentityFile="${SSH_PRIV_KEY_PATH}" -o StrictHostKeyChecking=no ${BASTION_SSH_USER}@${BASTION_IP} \
+            "for container in \$(sudo podman ps -q -a); do sudo podman rm \${container} || true; done"
+        echo "delete images"
+        ssh -o UserKnownHostsFile=/dev/null -o IdentityFile="${SSH_PRIV_KEY_PATH}" -o StrictHostKeyChecking=no ${BASTION_SSH_USER}@${BASTION_IP} \
+            "for image_url in \${image_list}; do sudo podman rmi \${image_url} || true; done"
+        echo "delete containers and images done"
+    done
+done
+
+# print the compelete result
+echo "The full result is: $(cat "${scan_result_file}" || true)"
+echo "The fips check returned succeeded with warnings for below csvs: $(cat "${scan_result_succeed_with_warnings_csvs}" || true)"
+echo "The csvs failed fips check are: $(cat "${scan_result_failure_csvs}" || true)"
+tar -czC "${data_dir}" -f "${ARTIFACT_DIR}/fips-check-soptional_operators-images-scan.tar.gz" . || true
+rm -rf "${data_dir}" || true
+
+# check how much time used
+show_time_used "$timestamp_start"
+
+echo "Generating the Junit for fips check optional operators scan"
+filename="fips-check-soptional_operators-images-scan"
+testsuite="fips-check-soptional_operators-images-scan"
+subteam="Security_and_Compliance"
+if $pass; then
+    cat >"${ARTIFACT_DIR}/${filename}.xml" <<EOF
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuite name="${testsuite}" failures="0" errors="0" skipped="0" tests="1" time="1">
```

Comment:
The test execution time is hard coded to 1. Refer to the other fips test to see how to avoid it.

Reply:
Updated. Details seen from https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/45225/rehearse-45225-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-proxy-fips-f14-fips-check/1739172421132357632/artifacts/aws-ipi-proxy-fips-f14-fips-check/fips-check-optional-operator-images-scan/artifacts/fips-check-soptional_operators-images-scan.xml